src/MeetingsSDKAdapter.js

```diff
@@ -165,18 +166,27 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
    * @param {string} ID ID to retrieve the SDK meeting object to add the local media to
    * @returns {Promise.<MediaStream>} Promise to requested media stream
    */
-  async getLocalMedia(ID) {
-    const localMedia = {localAudio: null, localVideo: null};
-
-    if (mediaSettings.sendAudio) {
-      localMedia.localAudio = await this.getStream(ID, {sendAudio: true});
-    }
-
-    if (mediaSettings.sendVideo) {
-      localMedia.localVideo = await this.getStream(ID, {sendVideo: true});
-    }
+  getLocalMedia(ID) {
+    const audio$ = this.getStream(ID, {sendAudio: true}).pipe(
```

Comment:
there is a test we forgot about: `if (mediaSettings.sendAudio)`

let's keep that feature:
```js
const audio$ = (
  mediaSettings.sendAudio ?
    this.getStream(ID, ...) :
    of({permission: null, stream: null})
  ).pipe(...);
```

same for video$